src/views/health-checks.vue

```diff
@@ -15,6 +15,25 @@
                     <td>{{CE.isDefined ? 'Yes': 'No'}}</td>
                 </tr>
             </table>
+            <hr />
+            
+            <h3>2. Common security check: blank-targetted links with no <code>rel="noopener"</code> (see <a href="https://developers.google.com/web/tools/lighthouse/audits/noopener" target="_blank" rel="noopner"> here</a>)</h3>
+            <p><button v-on:click="checkBlankTargettedLinks">Run check</button></p>
+            <table v-if="riskyAnchors.length > 0">
+                <tr>
+                    <th>HREF</th>
+                    <th>DOM Path</th>
+                    <th>innerText</th>
+                </tr>
+                <tr v-for="anchor in riskyAnchors" v-bind:key="anchor.name">
```

Comment:
Can you display all `a[target="_blank"]` and make use of CSS classes `ok-bg` and `error-bg` to highlight the ones without `noopener`?